### PR TITLE
Add build flavor for java packages

### PR DESCRIPTION
**Pull request by eabdullin1**

### CODE REVIEW

File: reference_data/platform_flavors.yaml [hunk 1]
```diff
@@ -389,3 +389,42 @@
       url: "https://build.almalinux.org/pulp/content/copr/soksanichenko-Pungi-almalinux-$releasever-src-dr/"
       debug: false
       production: false
+
+- name: java-portable
+  data:
```
Comment (javihernandez) [inline reference_data/platform_flavors.yaml:394]: Maybe the empty `data` field is causing the tests to fail?

Comment (eabdullin1) [reply reference_data/platform_flavors.yaml]: Fixed, but tests still failed

Comment (Korulag) [reply reference_data/platform_flavors.yaml]: No, that's not the issue. Looks like we were unable to construct coverage report:
"Error: TypeError: Cannot destructure property 'coverage' of 'report' as it is null.
Error: Cannot destructure property 'coverage' of 'report' as it is null."

**Comment by javihernandez** [comment]
right, not related, but tests are indeed failing, see:
```
File "/code/tests/fixtures/builds.py", line 6, in <module>
    from alws.crud.build import create_build, get_builds
  File "/code/alws/crud/build.py", line 20, in <module>
    from alws.schemas import build_schema
  File "/code/alws/schemas/build_schema.py", line 89, in <module>
    class BuildCreatePlatforms(BaseModel):
  File "pydantic/main.py", line 197, in pydantic.main.ModelMetaclass.__new__
  File "pydantic/fields.py", line 506, in pydantic.fields.ModelField.infer
  File "pydantic/fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line [75](https://github.com/AlmaLinux/albs-web-server/actions/runs/5059490268/jobs/9081098572?pr=559#step:6:76)8, in pydantic.fields.ModelField._type_analysis
  File "pydantic/fields.py", line 808, in pydantic.fields.ModelField._create_sub_type
  File "pydantic/fields.py", line 436, in pydantic.fields.ModelField.__init__
  File "pydantic/fields.py", line 552, in pydantic.fields.ModelField.prepare
  File "pydantic/fields.py", line 668, in pydantic.fields.ModelField._type_analysis
  File "/usr/lib64/python3.9/typing.py", line 852, in __subclasscheck__
    return issubclass(cls, self.__origin__)
TypeError: issubclass() arg 1 must be a class
```

**Pull request merged**